src/modal/modal-config.ts

```diff
@@ -11,6 +11,14 @@ export interface NgbModalOptions {
    */
   ariaLabelledBy?: string;
 
+  /**
+   * `aria-describedby` attribute value to set on the modal window.
+   *
+   * @since 6.0.4
```

Comment:
You shouldn't put this, because it is not clear when this will be actually merged.
We're usually adding this only for features, so for `X.X.0` releases just before doing the release.